### PR TITLE
Adding string implementation

### PR DESCRIPTION
**Pull request by karmoham**

### CODE REVIEW

File: coding-102-parsers/strings.py [hunk 1]
```diff
@@ -8,4 +8,143 @@
...
+   Locally configured policies:
+    route-map RMAP5 in
+    route-map RMAP6 out
+ Neighbor: 21.0.0.4, Address-Family: VPNv6 Unicast (VRF2)
+   Locally configured policies:
+    route-map RMAP7 in
+    route-map RMAP8 out
+    '''
+
+out = STRING
```
Comment (simingy) [inline coding-102-parsers/strings.py]: don't need to assign out to STRING
just do for line in STRING.splitlines()

File: coding-102-parsers/strings.py [hunk 2]
```diff
@@ -8,4 +8,143 @@
...
+            PYTHON_DICT['vrf'][vrf_name]['neighbor'][neighbor_name]\
+				['address_family'][address_family_name]['route_map_out'] = \
+					route_map_name
+
+        continue
+
+# print the parsed built structure
+print (PYTHON_DICT)
+
+PYTHON_DICT = {
```
Comment (simingy) [inline coding-102-parsers/strings.py]: this is a the expected output - comment it out so it's only for reference

File: coding-102-parsers/strings.py [hunk 3]
```diff
@@ -8,4 +8,143 @@
...
+					route_map_name
+        else:
+            PYTHON_DICT['vrf'][vrf_name]['neighbor'][neighbor_name]\
+				['address_family'][address_family_name]['route_map_out'] = \
+					route_map_name
+
+        continue
+
+# print the parsed built structure
+print (PYTHON_DICT)
```
Comment (simingy) [inline coding-102-parsers/strings.py]: import pprint; pprint.pprint()

**Review by simingy — CHANGES_REQUESTED** [review]

File: coding-102-parsers/strings.py (hunk 1 above)
Comment (karmoham) [reply coding-102-parsers/strings.py]: Done.

File: coding-102-parsers/strings.py (hunk 3 above)
Comment (karmoham) [reply coding-102-parsers/strings.py]: Done.

File: coding-102-parsers/strings.py (hunk 2 above)
Comment (karmoham) [reply coding-102-parsers/strings.py]: Done

**Review by simingy — APPROVED** [review]

**Pull request merged**